task_assign/task[] task_vect